jsk_topic_tools/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
   add_rostest(test/test_hz_measure.test)
   add_rostest(test/test_block.test)
 endif(NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
```

Comment:
newer than default hydro but older than current latest indigo.

It should be fixed the latest indigo catkin.